### PR TITLE
adding the code to accept multiple inputs for performance counters

### PR DESCRIPTION
**Pull request by gs185220**

### CODE REVIEW

File: cmd/check_performance_counter/cmd/root.go [hunk 1]
```diff
@@ -27,27 +31,47 @@ The defaults for this check have the --critical and --warning values set to 0, a
 retrieved is compared to be lesser than those values. Generally a counter value will be > 0, causing
 this check to emit an OK response when using these defaults.`,
 		Run: func(cmd *cobra.Command, args []string) {
-			cmd.ParseFlags(os.Args)
+			// cmd.ParseFlags(os.Args)
```
Comment (jk185160) [inline cmd/check_performance_counter/cmd/root.go:34]: This supposed to be commented out?

File: cmd/check_performance_counter/cmd/root.go [hunk 2]
```diff
@@ -27,27 +31,47 @@ The defaults for this check have the --critical and --warning values set to 0, a
...
 	rootCmd.MarkFlagRequired(counterNameFlag)
 	rootCmd.Flags().StringVarP(&metricName, "metric_name", "m", "", "the name of the metric generated by this check")
-	rootCmd.Flags().Float64VarP(&warning, "warning", "w", 0, "the threshold to issue a warning alert")
-	rootCmd.Flags().Float64VarP(&critical, "critical", "c", 0, "the threshold to issue a critical alert")
+	rootCmd.Flags().Float64VarP(&warning, "warning", "w", 0.0, "the threshold to issue a warning alert")
+	rootCmd.Flags().Float64VarP(&critical, "critical", "c", 0.0, "the threshold to issue a critical alert")
 	rootCmd.Flags().BoolVarP(&greaterThan, "greater_than", "g", false, "issue warnings if the metric is greater than the expected thresholds (default false)")
-	rootCmd.Flags().IntVarP(&pollingAttempts, "polling_attempts", "a", 2, "the number of times to fetch and average the performance counter")
-	rootCmd.Flags().IntVarP(&pollingDelay, "polling_delay", "d", 1, "the number of seconds to delay between polling attempts")
+	rootCmd.Flags().IntVarP(&pollingAttempty_cntrs, "polling_attempty_cntrs", "a", 2, "the number of times to fetch and average the performance counter")
```
Comment (jk185160) [inline cmd/check_performance_counter/cmd/root.go:73]: why are we renaming the variable here?  id love to leave the interface unchanged so I can avoid a major version upgrade

**Review by jk185160 — CHANGES_REQUESTED** [review]

File: cmd/check_performance_counter/cmd/root.go (hunk 2 above)
Comment (gs185220) [reply cmd/check_performance_counter/cmd/root.go:73]: This is actually done by mistake john, any change is not required in this line. I will update the PR

File: cmd/check_performance_counter/cmd/root.go (hunk 1 above)
Comment (gs185220) [reply cmd/check_performance_counter/cmd/root.go:34]: If we don't comment this, we got duplicates john.. attaching a screenshot for clear understanding.
image

**Review by jk185160 — APPROVED** [review]
LGTM. Thanks @gs185220

**Comment by jkerry** [comment]
fixes #190

**Pull request merged**